Sources/SyndiKit/Formats/Media/Podcast/PodcastSeason.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+public struct PodcastSeason: Codable {
+  public let name: String?
+
+  public let value: Int
```

Comment:
I would avoid `value` as a property name and describe what the `value` is. In this case 
>  The node value is an integer, and represents the season "number". It is required.

Therefore I would call this `number`